README.md

```diff
@@ -1,2 +1,81 @@
 # AndroidStreamComparison
-Comparison of streaming libraries for Android with low latency
+Comparison of streaming libraries for Android with low latency.
+## Goal
+Give an overview of existing protocols for live-streaming and libraries for android.
+Design a system of two smartphones, one of which transmits live video from its camera and the other receives it.
+## Overview
+### Review protocols
+The following protocols for stream transmission over the network are considered:
+-	Protocols work on TCP:
+     -   RTMP
+     -   HTTP and related HLS, DASH и MPEG-DASH
+-	Protocols work on UDP, implementing tcp reliability at the application layer:
+     -   SRT
+     -   RTMFP
+-   Protocol works on SCTP:
+    -   WebRTC
+-  Protocol works on RTP:
+    -   RTSP
+-   Protocols not included in the list:
+    -   FTL - the server-side implementation of Mixer’s FTL protocol is proprietary and closed-source as of this writing.
+
+Secure and reliable protocols will have inherently higher latency than UDP-based protocols. Therefore, we will only consider RTSP.
```

Comment:
WEBRTC? why is it out

Reply:
Because SCTP is the same as RTP, but with secrecy that will only slow down

Reply:
How can it be defined from text?